packages/grafana-ui/src/components/DateTimePickers/TimeRangePicker/TimePickerContent.tsx

```diff
@@ -179,26 +188,37 @@ export const TimePickerContentWithScreenSize: React.FC<PropsWithScreenSize> = (p
     <div id="TimePickerContent" className={cx(styles.container, className)}>
       <div className={styles.body}>
         {(!isFullscreen || !hideQuickRanges) && (
-          <CustomScrollbar className={styles.rightSide}>
-            {!isFullscreen && <NarrowScreenForm {...props} historyOptions={historyOptions} />}
-            {!hideQuickRanges && (
-              <>
-                <TimeRangeList
-                  title="Relative time ranges"
-                  options={quickOptions}
-                  onChange={onChangeTimeOption}
-                  value={timeOption}
-                />
-                <div className={styles.spacing} />
-                <TimeRangeList
-                  title="Other quick ranges"
-                  options={otherOptions}
-                  onChange={onChangeTimeOption}
-                  value={timeOption}
-                />
-              </>
-            )}
-          </CustomScrollbar>
+          <div className={styles.rightSide}>
+            <div className={styles.timeRangeFilter}>
+              <FilterInput width={0} value={searchTerm} onChange={setSearchQuery} placeholder={'Search quick ranges'} />
```

Comment:
Noticed that when you press clear, the entire time picker disappears:

![Kapture 2021-09-16 at 10 22 00](https://user-images.githubusercontent.com/2376619/133577427-0a21027b-4d82-475b-95b0-82f378282883.gif)

Reply:
One more nit: I would expect the search input to be autofocused when you open the time range picker so that you can start typing straight away :)

Reply:
Ah, I'll fix the clear and the focus, thanks

Reply:
https://github.com/grafana/grafana/pull/39073/commits/e7fa25eb068566500b465b76fa62d9ccf652c181
This should fix these issues